optimade/filtertransformers/elasticsearch.py

```diff
@@ -64,7 +65,7 @@ def __repr__(self):
 
 
 class Transformer(lark.Transformer):
```

Comment:
Would it cause any problems to rename this class? I started doing this in #287 but left in aliases to the old ones.

```suggestion
class ElasticsearchTransformer(lark.Transformer):
```